planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -2176,6 +2176,9 @@ void AvoidanceModule::addShiftPointIfApproved(const AvoidPointArray & shift_poin
     const size_t prev_size = path_shifter_.getShiftPointsSize();
     addNewShiftPoints(path_shifter_, shift_points);
 
+    // update ego velocity
+    ego_velocity_starting_avoidance_ = getEgoSpeed();
```

Comment:
@rej55  `addShiftPointIfApproved()` may be called long time before the avoidance, using the vehicle speed at this point to limit the acceleration does not make sense... does it?